metaflow/cli.py

```diff
@@ -234,8 +234,17 @@ def output_dot(obj):
 )
 @click.pass_obj
 def dump(obj, input_path, private=None, max_value_size=None, include=None, file=None):
-
     output = {}
+    
+    if file is not None and max_value_size is not None:
+        raise CommandException("max_value_size set to None when a file variable is provided will get replace the file variable")
```

Comment:
can you clarify what is the intention here?

Reply:
Going through previous reviews of this work I was addressing a comment here https://github.com/Netflix/metaflow/pull/806#discussion_r742255016 

I do not have any emotional attachment here so we can remove this and keep a single conditional statement.

Reply:
Will wait to hear back on this issue before pushing the changes I made above